src/components/FlowRunCreateForm.vue

```diff
@@ -125,7 +126,7 @@
   }
 
   const combinedParameters = computed(() => {
-    return mapper.map('SchemaValuesResponse', { values: { ...props.deployment.rawParameters, ...props.parameters }, schema: props.deployment.parameterOpenApiSchema }, 'SchemaValues')
+    return mapper.map('SchemaValuesResponse', { values: merge({}, props.deployment.rawParameters, props.parameters), schema: props.deployment.parameterOpenApiSchema }, 'SchemaValues')
```

Comment:
This is probably a safer way to merge these

Reply:
+1